src/app/types/finalityProviders.ts

```diff
@@ -16,3 +16,18 @@ export interface Description {
   securityContact: string;
   details: string;
 }
+
+export type FinalityProviderState = keyof typeof stateMap;
+
+const stateMap = {
+  FINALITY_PROVIDER_STATUS_INACTIVE: "Inactive",
+  FINALITY_PROVIDER_STATUS_ACTIVE: "Active",
+  FINALITY_PROVIDER_STATUS_JAILED: "Jailed",
+  FINALITY_PROVIDER_STATUS_SLASHED: "Slashed",
+};
+
+export const mapFinalityProviderState = (
+  state: FinalityProviderState,
+): string => {
+  return stateMap[state];
+};
```

Comment:
@jeremy-babylonlabs `type` files should not contain instances or implementations. Move this logic to `FinalityProvider` component
```suggestion
export type FinalityProviderState = "FINALITY_PROVIDER_STATUS_INACTIVE" | "FINALITY_PROVIDER_STATUS_ACTIVE " | "  FINALITY_PROVIDER_STATUS_JAILED" | "FINALITY_PROVIDER_STATUS_SLASHED" ;
```